app/styles/components/_offering-editor-learnergroups.scss

```diff
@@ -0,0 +1,40 @@
+@mixin subgroup-style {
```

Comment:
This should be styled using the existing _tree-list.scss.  That file can be modified, but we want to use the same visual style everywhere this UX exists.